test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -47,6 +47,18 @@ describe('distribute', function () {
       ['(-2x^2 * 3x - 2x^2 * -4)',
         '(-6x^3 + 8x^2)']
     ],
+    ['(x - 2)^2',
```

Comment:
I think some more tests would be good. What if there are more than 2 terms? What if we raise to a power that is not 2?